circe-display-images.el

```diff
@@ -74,7 +74,7 @@ See `enable-circe-display-images'."
   :group 'circe)
 
 (defcustom circe-display-images-image-regex
-  "\\(https?://[^ ]*?\\.\\(?:png\\|jpg\\|jpeg\\|svg\\|gif\\)\\)"
+  "\\(https?://[^ ]*?\\.\\(?:png\\|jpg\\|jpeg\\|svg\\|gif\\)[^\]\)>\"\' \n]*\\)"
```

Comment:
The reason you use rx is because it generates optimal regex and avoids typical errors, such as needlessly escaping characters that are not considered special in a character alternative. The URL delimiters would be matched by `(rx (* (not (any "<>\"\t\r\n "))))` for example. Parentheses, square brackets and single quotes are not supposed to be there.

Additionally, that regex should be used after the "https?://" part. rx allows avoiding duplication here if the non-macro form is used.

Reply:
I see, I pushed my fixes to the existing regex that I already saved before working on replacing it with rx.